src/library/scala/collection/View.scala

```diff
@@ -228,9 +196,9 @@ object View extends IterableFactory[View] {
   }
 
   @SerialVersionUID(3L)
-  class RightPartitionedWith[A, A1, A2](partitionWith: PartitionWith[A, A1, A2], f: A => Either[A1, A2]) extends AbstractView[A2] {
+  class RightPartitionedWith[A, A1, A2](val underlying: SomeIterableOps[A], f: A => Either[A1, A2]) extends AbstractView[A2] {
```

Comment:
Any reason for `underlying`  being public?

Reply:
@allanrenucci `underlying` is not being publicly accessed via this class, so I've removed the explicit `val` modifier now. Added `val` earlier keyword cause the `underlying` was marked with `val` from calling method, and I didn't know it also causes the field to be publicly accessible.